mirror/images.txt

```diff
@@ -8,7 +8,7 @@
 
 # ceph-csi devel
 docker.io/rook/ceph:v1.12.1      rook/ceph:v1.12.1
-quay.io/ceph/ceph:v18            quay.io/ceph/ceph:v18
+quay.io/ceph/ceph:v18            ceph/ceph:v18
```

Comment:
@nixpanic why this is only required for v18 not for any other versions?

Reply:
I think I manually pusehed the other tags without `quay.io`, so they work fine. Ideally everything is automated and no manual pushing is needed anymore (in the future).